proto/brpb.proto

```diff
@@ -731,14 +731,19 @@ message DataFileInfo {
 
     // The original region range the log file belongs to.
     // In older versions, this might be empty.
+    // This may not be accurative: file may contain keys less than this.
     bytes region_start_key = 20;
     // The original region range the log file belongs to.
     // In older versions, this might be empty.
+    // This may not be accurative: file may contain keys greater than this.
     bytes region_end_key = 21;
     // The region epoch that the log file belongs to.
     // In older versions, this might be empty.
-    // If a region get split or merged during observing,
-    // the file may contain multi epoches.
+    // If a region get split or merged during observing, the file may contain multi epoches.
+    // This may not be complete: file may contain records from other versions.
+    //
+    // For now(v8.5), there will only be exactly one epoch due to the restriction of log backup code.
```

Comment:
you mean from v8.5 on?

Reply:
I guess perhaps we'd better don't mention the version, as it isn't merged to master.🤔